projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -0,0 +1,231 @@
+# Reuse places & organizers
+
+When creating an event via the UiTdatabank Entry API, you can and must use existing places (locations) and organizers in UiTdatabank.
+
+By reusing existing places and organizers you save yourself work, as you don't have to create the place or organizer yourself. In addition, you ensure that no duplicates are created in UiTdatabank and that other integrators can also use the same unique and clean dataset.
+
+Only in the case a certain place or organizer does not exist yet, a new one can be created.
+
+<!-- theme: warning -->
+
+> When sending event content to UiTdatabank trough the Entry API it is essential to reuse existing organizer IDs and place IDs. Otherwise your integration can be blocked because of the creation of duplicates.
+
+On this page, we outline some options to reuse existing places & organizers from UiTdatabank in your integration.
+
+## Reuse existing places
+
+### Storing the place ID(s) inside your application
+
+This is the simplest method. You store the identifier(s) of the place(s) inside your application.
```

Comment:
```suggestion
This is the simplest method. You store the URL(s) of the place(s) inside your application.
```

Most integrators will need the place URL instead of the ID to create/update events, as the `location.@id` must be an URL. Only in this endpoint would they specifically need the ID: https://docs.publiq.be/docs/uitdatabank/a6ad6b7450dc2-location-update (but when using `PUT /events/{eventId}` to update the location they will still need the URL)